content/MobileAppEmpty-CSharp/GitIgnore

```diff
@@ -45,4 +45,10 @@ Certificate/
 .droidres/
 
 secrets.json
-Secrets.cs
\ No newline at end of file
+Secrets.cs
+
+# Auto generated files
+project.lock.json
+Company.MobileApp.Droid.nuget.props
+Company.MobileApp.iOS.nuget.props
```

Comment:
This currently wouldn't also prevent checking in `Company.MobileApp.Platform.nuget.targets`.
Please consolidate this down to Company.MobileApp.*nuget.*